installation/resources/velero-secret.yaml.tpl

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
```

Comment:
@a-thaler is right. We want to get rid of all such template files, otherwise users must use `sed` which isn't really user-friendly.
The proper way to do it is to create configmap with overrides explicitly. Here's an example:
```
kubectl create configmap velero-credentials-overrides -n kyma-installer \
  --from-literal=configuration.provider=${CLOUD_PROVIDER} \
  --from-literal=configuration.volumeSnapshotLocation.bucket=${BSL_BUCKET} \
&& kubectl label configmap velero-credentials-overrides -n kyma-installer installer=overrides component=velero
```
An example of such command along with description of all available parameters should be placed in the component documentation

Reply:
As discussed, we will take of it in a separate story #4212